build_deploy.sh

```diff
@@ -4,7 +4,7 @@ TAG=$(git rev-parse --short=7 HEAD)
 
 BM_INVENTORY_IMAGE="quay.io/app-sre/bm-inventory"
 
-docker build -t "${BM_INVENTORY_IMAGE}:latest" -f Dockerfile.demo .
+docker build -t "${BM_INVENTORY_IMAGE}:latest" -f Dockerfile.bm-inventory .
```

Comment:
Last time I checked, docker build would fail with that Dockerfile.
Maybe also include the docker build command in the `pr_check.sh` just to validate it?